### PR TITLE
Update acq report format a bit, tidy code, change raise_exc default

### PR DESCRIPTION
**Pull request by taldcroft**
Changing the default of `raise_exc` is the big API change here.

### CODE REVIEW

**Comment by taldcroft** [comment]
Hmm, I don't see a way to change the branch this merges into.  Gotta run, but I guess will make a new PR for merge to master?

**Comment by jeanconn** [comment]
For me, I can change target branches from the edit (that looks like it is for the subject, but gives a drop down for target branch).  Is that what you meant?

**Comment by taldcroft** [comment]
But of course!

File: proseco/tests/test_catalog.py [hunk 1]
```diff
@@ -357,7 +357,8 @@ def test_t_ccd_effective_acq_guide(t_ccd_case):
 
 
 def test_call_args_attr():
-    aca = get_aca_catalog(**mod_std_info(optimize=False, n_guide=0, n_acq=0, n_fid=0))
+    aca = get_aca_catalog(**mod_std_info(optimize=False, n_guide=0, n_acq=0, n_fid=0),
+                          raise_exc=False)
```
Comment (jeanconn) [inline proseco/tests/test_catalog.py]: Not sure about this one.

Comment (taldcroft) [reply proseco/tests/test_catalog.py]: This is reproducing the original default, and this test requires `raise_exc` to pass.

Comment (taldcroft) [reply proseco/tests/test_catalog.py]: I meant it needs `raise_exc=False` to pass.

Comment (jeanconn) [reply proseco/tests/test_catalog.py]: I suppose it really just needs raise_exc to *do* the test on call_args.

**Pull request merged**